src/javascript/app/pages/user/metatrader/metatrader.config.js

```diff
@@ -223,7 +223,7 @@ const MetaTraderConfig = (() => {
                 response.binary_transaction_id,
             ]),
             prerequisites: () => new Promise((resolve) => {
-                if (Client.get('is_virtual')) {
+                if (Client.get('is_virtual') || /^MX/i.test(Client.get('loginid'))) {
```

Comment:
can you check by iom landing company shortcode instead of hardcoding broker code?